integration/system/options_test.go

```diff
@@ -0,0 +1,24 @@
+package system
+
+import (
+	"net/http"
+	"testing"
+
+	req "github.com/docker/docker/internal/test/request"
+	"gotest.tools/assert"
+)
+
+func TestAPIOptionsRoute(t *testing.T) {
+	defer setupTest(t)()
+
+	resp, _, err := req.Do("/", req.Method(http.MethodOptions))
+	assert.NilError(t, err)
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+}
+
+func TestAPIGetEnabledCORS(t *testing.T) {
+	resp, body, err := req.Get("/version")
```

Comment:
This could be changed to `/_ping` perhaps (as we're not interested in the `version` output, just in the headers for any response)